src/autorest-core/lib/pipeline/commonmark-documentation.ts

```diff
@@ -61,7 +61,7 @@ export async function ProcessCodeModel(codeModel: DataHandleRead, scope: DataSto
       const nodeOriginal = CloneAst(node);
       const key = nodeOriginal.key.value;
       const origKey = key + "_Original";
-      nodeOriginal.key.value = origKey
+      nodeOriginal.key.value = origKey;
```

Comment:
that's exactly the tool I use myself, but its "auto-fixes" sometimes break stuff, otherwise I'd be for that!